livre/Eq/Channel.cpp

```diff
@@ -295,10 +296,25 @@ class Channel
         livre::Node* node = static_cast< livre::Node* >( _channel->getNode( ));
         livre::Window* window = static_cast< livre::Window* >( _channel->getWindow( ));
         livre::Pipe* pipe = static_cast< livre::Pipe* >( window->getPipe( ));
+
+
+        const uint32_t minDepth = pipe->getFrameData()->getVRParameters()->minDepth;
```

Comment:
get VRParameters and dashTree as references/shortcuts. They are used multiple times here